Documentation/network/servicemesh/gateway-api/host-network-mode.rst

```diff
@@ -0,0 +1,70 @@
+Host network mode
+#################
+.. note::
+    Supported since Cilium 1.16+
+
+This mode allows you to expose the Cilium GatewayAPI Gateway
+directly on the host network.
+This is useful in some edge cases where one doesn't want, or can't, use
+LoadBalancer type services (dev environments, cluster-external
+loadbalancer, ...).
+
+The host network mode can be enabled via Helm flag
+``gatewayAPI.hostNetwork.enabled=true``.
+Once enabled, the host network ports can be specified on the ``Gateway``
+resources via ``spec.listeners.port``. The port must be unique per ``Gateway``
+resource and higher than ``1023``.
+
+.. warning::
+    * Enabling the Cilium GatewayAPI host network mode automatically disables the LoadBalancer/NodePort type service mode. They are mutually exclusive.
+    * The listener is exposed on all interfaces (``0.0.0.0`` for IPv4 and/or ``::`` for IPv6).
+    * Be aware that misconfiguration might result in port clashes. Configure unique ports that are still available.
```

Comment:
> Configure unique ports that are still available.

... that are still available on all Cilium Nodes where Gateway API listeners are exposed.